### ISSUE TITLE
Setting ryuk.container.timeout causes a ClassCastException

### ISSUE THREAD

**Issue by jamieswogger** [testcontainers/testcontainers-java#684]
Tried setting `ryuk.container.timeout` by creating a `testcontainers.properties` that contains the following:
```
ryuk.container.timeout=120
```

This is successfully picked up, but when [TestcontainersConfiguration#getRyukTimeout](https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java#L54) is called is calls `properties.getOrDefault`, which returns a string "120" which is attempted to be cast to an `Integer`. This results in the following error:

```
java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
```

A fix for this is to change [TestcontainersConfiguration#getRyukTimeout](https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java#L54) to
```
public Integer getRyukTimeout() {
    return Integer.parseInt((String) properties.getOrDefault("ryuk.container.timeout", "30"));
}
```

**Comment by jamieswogger** [comment]
I've created a pull request with the proposed fix. If that PR is approved and merged is it possible to release a patch soon with this fix?

https://github.com/testcontainers/testcontainers-java/pull/685

**Comment by bsideup** [comment]
Released as 1.7.3
